libs/langchain/langchain/chains/graph_qa/hugegraph.py

```diff
@@ -88,7 +88,7 @@ def _call(
         )
 
         result = self.qa_chain(
-            {"question": question, "context": context},
+            {"question": question, "context": context, "sql": generated_gremlin},
```

Comment:
the qa_chain is just mean to answer the question given the retrieved data, why include the sql query in that?

Reply:
> the qa_chain is just mean to answer the question given the retrieved data, why include the sql query in that?

because it is possible that when the generated SQLhas a problem, it could result in no data . At this time, the user will want to view and check if the generated SQL is correct